llvm-hs/src/LLVM/Internal/FFI/Builder.hs

```diff
@@ -97,11 +97,26 @@ foreign import ccall unsafe "LLVMBuildArrayAlloca" buildAlloca ::
 foreign import ccall unsafe "LLVM_Hs_BuildLoad" buildLoad' ::
   Ptr Builder -> LLVMBool -> Ptr Value -> MemoryOrdering -> SynchronizationScope -> CUInt -> CString -> IO (Ptr Instruction)
 
+buildLoad :: Ptr Builder
```

Comment:
Please keep these signatures consistent with the other signatures. I’m open to changing the style here in general but let’s keep that separate from this PR and changing half of them is weird.